build.sbt

```diff
@@ -53,6 +63,10 @@ lazy val fixedWidth = (project in file("."))
     },
     libraryDependencies ++= baseDependencies,
   )
+  .settings(
+    jacocoReportSettings := commonJacocoReportSettings.withTitle("fixed-width Jacoco Report"),
```

Comment:
This is a single module project, so it makes more sense to me to have the whole settings object defined in a single place.